src/lib/mockserver.js

```diff
@@ -139,6 +118,47 @@ class MockServer {
             }
         }
     }
+
+    /**
+     * Spawn docker logs until mocks are set up
+     */
+    async _dockerSpawnLogs() {
+        debug(`> docker logs -f ${this.container}`);
+        const proc = spawn('docker', ['logs', '-f', this.container]);
+        let isRunning;
+
+        proc.stdout.on('data', function(data) {
+            const stdout = data.toString();
+            if (stdout && stdout.includes('started on ports: [80, 443]')) {
+                debug("Stdout", stdout);
```

Comment:
Minor: maybe log for every line (not just matching?), otherwise not much to see for debug purposes. And prefix with "mock-server:" or nothing at all.